### PR TITLE
Flesh out mocks for frontend

### LINKED ISSUE #13: Create basic frontend structural mocks

### PR DESCRIPTION
**Pull request by Graywing13**
- Add basic (placeholder) content for sidebar / timeline / cards / map
- Fix sidebar animation
- Merge sidebarSlice into viewSlice, to keep track of how the app looks in one place

Closes #13

### CODE REVIEW

File: client/src/redux/reducers/viewSlice.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      state.isSidebarOpen = !state.isSidebarOpen
+    },
+    openSidebar: (state) => {
+      state.isSidebarOpen = true
+    },
+    closeSidebar: (state) => {
+      state.isSidebarOpen = false
+    },
+  },
+})
```
Comment (L0Lmaker) [inline client/src/redux/reducers/viewSlice.js]: If these are going to get longer, we should perhaps extract functions out of them and reference the function name.

File: client/public/globe.png
Comment (L0Lmaker) [inline client/public/globe.png:1]: nit: Are there public urls we can use for the pictures? 
since we will be removing them eventually.

**Review by L0Lmaker — APPROVED** [review]
LGTM as a base, we will need to update the design. We could use MUI to standardize it.

File: client/public/globe.png
Comment (Graywing13) [reply client/public/globe.png:1]: I tried but it didn't work, mayb i was doing smth wrong at first

Comment (Graywing13) [reply client/public/globe.png:1]: And there's no public URL for the globe. I can remove the images later after we add the real deal :)

File: client/src/redux/reducers/viewSlice.js (hunk 1 above)
Comment (Graywing13) [reply client/src/redux/reducers/viewSlice.js]: Good idea, done

**Pull request merged**